geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestCommand.scala

```diff
@@ -79,6 +79,9 @@ object IngestCommand {
       "to override file extension recognition")
     var format: String = null
 
+    @Parameter(names = Array("-skH", "--skip-header"), description = "flag to skip the first line (header) of a csv/tsv data file")
```

Comment:
for the short version we decided to not use any camel case (all lower case args) perhaps this would be better to leave out ```-skH``` or make it ```-skh```